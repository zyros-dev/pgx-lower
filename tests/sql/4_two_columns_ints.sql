LOAD 'pgx_lower.so';
-- CRITICAL: Do not change logging level - required for MLIR pipeline visibility
-- NOTICE level enables full debugging of PostgreSQL AST → RelAlg → DB → LLVM → JIT pipeline
-- WARNING level suppresses essential MLIR compilation logs and breaks debugging capability
SET client_min_messages TO NOTICE;

DROP TABLE IF EXISTS test;

CREATE TABLE test (
    id SERIAL PRIMARY KEY,
    col2 INTEGER
);

INSERT INTO test(col2)
SELECT generate_series(1, 100);

SELECT * FROM test LIMIT 5;
