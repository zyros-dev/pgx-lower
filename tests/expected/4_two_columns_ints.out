LOAD 'pgx_lower.so';
NOTICE:  Installing custom executor hook...
NOTICE:  Registering custom sigsegv handler!
SET client_min_messages TO WARNING;
DROP TABLE IF EXISTS test;
CREATE TABLE test (
    id SERIAL PRIMARY KEY,
    col2 INTEGER
);
INSERT INTO test(col2)
SELECT generate_series(1, 100);
SELECT * FROM test LIMIT 5;
 id | col2 
----+------
  1 |    1
  2 |    2
  3 |    3
  4 |    4
  5 |    5
(5 rows)

